src/main/kotlin/no/nav/syfo/metrics/MetricRegister.kt

```diff
@@ -9,3 +10,21 @@ val HTTP_HISTOGRAM: Histogram = Histogram.Builder()
     .name("requests_duration_seconds")
     .help("http requests durations for incoming requests in seconds")
     .register()
+
+val EGENMELDT_SYKMELDING_COUNTER: Counter = Counter.build()
+        .namespace(METRICS_NS)
+        .name("egenmeldt_sykmelding_count")
+        .help("Number of egenmeldt sykmelding requests")
+        .register()
+
+val EGENMELDT_SYKMELDING_PERMUTED_COUNTER: Counter = Counter.build()
```

Comment:
Dette skjønner ikke jeg hva betyr :) Men trenger vi den egentlig?

Reply:
Det er antall egenmeldte sykmeldinger * arbeidsgivere i forespørselen. Typ, jeg sykmelder meg og har tre arbeidsgivere, så da blir denne inkrementert tre ganger. Om vi trenger det vet jeg ikke, men det er et hakket mer presist tall enn å bare telle antall sykmeldingrequests som kommer inn.

Reply:
Ah sånn ja, da er jeg enig i at den er nyttig :) Men kanskje den kan få et annet navn? Evt om denne kan være "EGENMELDT_SYMELDING_COUNTER", og så kanskje den andre kan hete noe med antall requests?

Reply:
Yep!

Reply:
Ser at du allerede hadde laget en EGENMELDT_SYKMELDING_REQ_COUNTER som bør dekke behovet :) Da tenker jeg vi kan fjerne en av metrikkene slik at vi har en for antall requests og en for antall sykmeldinger som opprettes :)

Reply:
Ok! :)